services/src/main/java/org/keycloak/social/twitter/TwitterIdentityProvider.java

```diff
@@ -190,8 +191,10 @@ public Response authResponse(@QueryParam("state") String state,
 
             AuthenticationSessionModel authSession = null;
             try (VaultStringSecret vaultStringSecret = session.vault().getStringSecret(getConfig().getClientSecret())) {
-                Twitter twitter = new TwitterFactory().getInstance();
+                ConfigurationBuilder builder = new ConfigurationBuilder();
+                builder.setIncludeEmailEnabled(true);
 
+                Twitter twitter = new TwitterFactory(builder.build()).getInstance();
```

Comment:
@NathanStrobbe this could be even shortned to one line: ` Twitter twitter = new TwitterFactory(new ConfigurationBuilder().setIncludeEmailEnabled(true).build()).getInstance();`